app/Http/Controllers/SignUpController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use DB;
+
+
+class SignUpController extends Controller
+{
+    
+    public function design(Request $request){
```

Comment:
let's try to make relevant naming functions, for example Create Page, function name should be create(). In programming naming is important so that developers can understand the purpose and the usage of that function.